.github/workflows/unit_tests.yml

```diff
@@ -25,17 +25,25 @@ jobs:
       uses: actions/setup-node@v1.4.2
       with:
         node-version: 12
-    - name: Set up Python environment${{ matrix.python-version }}
+    - name: Set up Miniconda Python environment
       uses: goanpeca/setup-miniconda@v1.4.1
       with:
-        environment-file: mne-bids/environment.yml
+        environment-file: environment.yml
         activate-environment: mne_bids
         auto-update-conda: true
         channel-priority: strict
         channels: conda-forge, defaults
+    - name: Install mne-python
+      shell: bash -l {0}
+      run: |
+        pip uninstall --yes mne-python
```

Comment:
```suggestion
        pip uninstall --yes mne
```

Reply:
Actually I believe we don't need this line at all, will remove